sc/social/like/browser/viewlets.py

```diff
@@ -68,4 +71,20 @@ class SocialLikesViewlet(BaseLikeViewlet):
     """Viewlet used to display the buttons
     """
     render = ViewPageTemplateFile('templates/sociallikes.pt')
-    render_method = 'plugin'
+
+    @property
+    def render_method(self):
+        tools = getMultiAdapter((self.context, self.request),
```

Comment:
This will be the ''only'' place where plone.api is used. Adding this dependency before #44 will be implemented isn't too much?